config/manager/manager.yaml

```diff
@@ -57,7 +57,7 @@ spec:
       #             values:
       #               - linux
       securityContext:
-        runAsNonRoot: true
+        runAsNonRoot: false
```

Comment:
I see this error otherwise
```
  Warning  Failed       34s (x9 over 2m12s)    kubelet            Error: container has runAsNonRoot and image will run as root (pod: "instaslice-operator-controller-manager-6bc4ccb79f-hvtm6_instaslice-system(1d6296ae-d680-4cc5-a7b8-606fff2f8ab3)", container: manager)
```